### ISSUE TITLE
Cannot install llama-cpp-python

### ISSUE THREAD

**Issue by donnadulcinea** [imartinez/privateGPT#1252]
Working in Ubuntu 22.04.3 LTS, and trying to install PrivateGPT I get the error.
Did someone else succeed to address it?

```
dd@ai:~/data/privateGPT$ poetry install --with ui,local
Installing dependencies from lock file

Package operations: 1 install, 0 updates, 0 removals

  • Installing llama-cpp-python (0.2.17): Failed

  LinkOutsideDestinationError

  'llama_cpp_python-0.2.17/vendor/llama.cpp/spm-headers/ggml.h' would link to '/tmp/ggml.h', which is outside the destination

  at /usr/lib/python3.10/tarfile.py:806 in _get_filtered_attrs
       802│             if os.path.isabs(member.linkname):
       803│                 raise AbsoluteLinkError(member)
       804│             target_path = os.path.realpath(os.path.join(dest_path, member.linkname))
       805│             if os.path.commonpath([target_path, dest_path]) != dest_path:
    →  806│                 raise LinkOutsideDestinationError(member, target_path)
       807│     return new_attrs
       808│
       809│ def fully_trusted_filter(member, dest_path):
       810│     return member

Cannot install llama-cpp-python.

```

Note that: `gcc-11 is already the newest version (11.4.0-1ubuntu1~22.04).`

**System:**
 - OS: Ubuntu 22.04.3 LTS

**Comment by MKdir98** [comment]
I'm getting this too for a week.

**Comment by lopagela** [comment]
Try to install `llama-cpp-python` outside poetry (using `pip`).

If the error persists, try to fix `llama-cpp-python` to a previous version.

**Comment by DivineMK** [comment]
Revert to poetry 1.6.0 works for me as well as stated at https://github.com/abetlen/llama-cpp-python/issues/905#issuecomment-1812389274

**Comment by lopagela** [comment]
From what I see, this will be fixed in poetry `1.7.1` (that will be released very soon): https://github.com/python-poetry/poetry/pull/8667

**Comment by lopagela** [comment]
Fixed with poetry `1.7.1` -- this issue was specific to poetry `1.7.0`
